README.md

```diff
@@ -1,2 +1,6 @@
 # hello-world
-"The Hello World project is a time-honored tradition in computer programming." - GitHub This exercise will get us started with collaboration using GitHub!
+
+"The Hello World project is a time-honored tradition in computer programming." - GitHub
+
+This exercise will get us started with collaboration using GitHub!
```

Comment:
Well, with some in-browser options and features of GitHub; but this will do just fine.